LICENSE

```diff
@@ -189,3 +189,32 @@ third-party archives.
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+ADDENDUM
```

Comment:
Can we move this part (or at least a few words) up to a preface at the top? It's easy to miss addendums